src/ECharts/Options.purs

```diff
@@ -0,0 +1,130 @@
+module ECharts.Options (
+  Option(..),
+  optionDefault,
+  setOption
+  ) where
+
+import Control.Monad.Eff
+import Data.Maybe
+import Data.Function
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+import Data.Argonaut.Combinators
+import Data.StrMap (fromList)
+
+import ECharts.Chart
+import ECharts.Color
+import ECharts.Series
+import ECharts.Timeline
+import ECharts.Toolbox
+import ECharts.Tooltip
+import ECharts.Title
+import ECharts.Legend
+import ECharts.DataRange
+import ECharts.DataZoom
+import ECharts.RoamController
+import ECharts.Grid
+import ECharts.Axis
+import ECharts.Utils
+import ECharts.Effects
+
+{- To set second series tooltip
```

Comment:
That's just when updating, right? You can set multiple series all at once, right?